Commands/Commands/queue.js

```diff
@@ -426,10 +426,9 @@ commands.registerVote = {
                   }
                 }).catch(e => {
                   if (e.statusCode === 404) {
```

Comment:
[`422 Unprocessable Entity`](https://httpstatuses.com/422) is used to indicate the voting being closed as wel as 404